Projects/Contracts/Sources/Reusables/ContractRow.swift

```diff
@@ -130,7 +100,7 @@ extension ContractRow: Reusable {
                 contentView.layer.zPosition = .greatestFiniteMagnitude
 
                 if !UITraitCollection.isCatalyst {
-                    contentView.hero.id = "contentView_\(self.contract.id)"
+                    contentView.hero.id = "contentView_\(self.contract.displayName)"
```

Comment:
What happens if you have two contracts with the same displayName, isn't it better to use the ID?